### PR TITLE
S12.4 verify dataline directory creation fix

### PR DESCRIPTION
**Pull request by anthony2261**
*Issue #, if available:*
#141 

*Description of changes:*
- Fix packaging github action to use the new location of `main.py` (under `dataline/`) and drop `llama_index`
- Remove reference to static `conn` object and obsolete functions so that the `user_data_dir` gets properly created on App startup

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: backend/dataline/context_builder.py [hunk 1]
```diff
@@ -22,148 +23,148 @@
...
+
+#                 table_names = [s.strip().replace("'", "").replace('"', "") for s in str(response).strip().split(",")]
+#                 if any(table_name not in self.sql_database.get_table_names() for table_name in table_names):
+#                     logger.info("Invalid table names: Reasking failed - continuing anyway")
+
+#         context_str = ""
+#         for table_name in table_names:
+#             context_str += self.full_context_dict[table_name.strip()] + "\n"
+
+#         if store_context_str:
```
Comment (RamiAwar) [inline backend/dataline/context_builder.py:167]: Just delete it 😁

File: backend/dataline/main.py [hunk 2]
```diff
@@ -122,8 +122,9 @@ async def execute_sql(
 
 @app.get("/toggle-save-query/{result_id}")
 async def toggle_save_query(result_id: str) -> SuccessResponse[None]:
-    db.toggle_save_query(result_id=result_id)
-    return SuccessResponse()
+    with db.DatabaseManager() as conn:
+        db.toggle_save_query(result_id=result_id, conn=conn)
+        return SuccessResponse()
```
Comment (RamiAwar) [inline backend/dataline/main.py:127]: Lets change this to use repos while we're at it. Time to take out allthis shit code

File: backend/dataline/old_services.py [hunk 3]
```diff
@@ -36,79 +34,79 @@ def run_sql(self, sql: str):
...
+#             self.query_manager.query(query, table_context=context_str, message_history=message_history)
+#         )
+#         data = json.loads(generated_json)
+#         result = SQLQueryResult(**data, selected_tables=table_names)
+
+#         if result.sql:
+#             # Validate SQL
+#             valid, error = self.sql_db.validate_sql(result.sql)
+#             if not valid:
+#                 logger.debug("\n\n------------------\n\n")
```
Comment (RamiAwar) [inline backend/dataline/old_services.py:91]: delete

**Review by RamiAwar — CHANGES_REQUESTED** [review]

File: backend/dataline/main.py (hunk 2 above)
Comment (anthony2261) [reply backend/dataline/main.py:127]: Upon further investigation, I noticed that "toggle_save_query" is not used anywhere on the frontend, and the table `SavedQueryModel` is obsolete. Should I remove it and anything related to it?

Comment (RamiAwar) [reply backend/dataline/main.py:127]: Oh.. right I removed this a while back in favor of just storing edits.

We're going to add it later when we do dashboards and saved queries and stuff. But yeah just remove it, we will find it in the history when it's time to reimplement

**Review by RamiAwar — APPROVED** [review]

**Pull request merged**